docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -620,13 +620,13 @@ It means that you are trying to convert a topology contains the `_contrib_box_nm
 
 #### 99. What does the message "ModelOptimizer is not able to parse *.caffemodel" mean? <a name="question-99"></a>
 
-If a `*.caffemodel` file exists and is correct, the error occurred possibly because of the use of Python protobuf implementation. In some cases, error messages may appear during model parsing, for example: "`utf-8` codec can't decode byte 0xe0 in position 4: invalid continuation byte in field: mo_caffe.SpatialTransformerParameter.transform_type". You can either use Python 3.6/3.7 or build the `cpp` implementation of `protobuf` yourself for your version of Python. For the complete instructions about building `protobuf` from sources, see the appropriate section in the [Converting Models with Model Optimizer](../Deep_Learning_Model_Optimizer_DevGuide.md) guide.
+If a `*.caffemodel` file exists and is correct, the error occurred possibly because of the use of Python protobuf implementation. In some cases, error messages may appear during model parsing, for example: "`utf-8` codec can't decode byte 0xe0 in position 4: invalid continuation byte in field: mo_caffe.SpatialTransformerParameter.transform_type". You can either use Python 3.7 or build the `cpp` implementation of `protobuf` yourself for your version of Python. For the complete instructions about building `protobuf` from sources, see the appropriate section in the [Converting Models with Model Optimizer](../Deep_Learning_Model_Optimizer_DevGuide.md) guide.
```

Comment:
Did we already mention about Python 3.6 support drop in the previous release?
I think we need to make a note and notify users about it and only after this is allowed to remove it overall.

Reply:
EXACTLY!
In the release-notes "https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes-2021.html" Ubuntu 18.04 is moved to "supported with limitations", but nothing said about deprecation of Python 3.6.
Also the latest release-notes don't announce neither the deprecation nor removal of Python v3.6.
As long as Ubuntu 18.04LTS is supported Python v3.6 "still exists".

Reply:
Good comments, thank you for this!
We will discuss it internally on one of next meetings.
Decision and consequences to stay with support of py3.6 is a little bit more complicated, so we are taking each point of view to our considerations carefully.